frontend/src/i18n/locales/en-us.json

```diff
@@ -227,7 +227,7 @@
     "received": "Received",
     "withdrawn": "Withdrawn",
     "hide": "Hide",
-    "withdraw": "withdraw",
+    "withdraw": "Withdraw",
```

Comment:
@apbendi I pushed this change which ensures the yellow "Withdraw" button in this table starts with a capital W

<img width="818" alt="image" src="https://user-images.githubusercontent.com/17163988/170788435-f47d28fd-d6ac-49fa-ba55-5e737a9daf39.png">